app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,12 @@
  *= require_tree .
  *= require_self
  */
+.room {
```

Comment:
Cool styles! (Just FYI, we won't ask you for styling on the finals or any other ICs, this is just for practice 👍 )